consensus/polybft/fsm.go

```diff
@@ -425,27 +443,33 @@ func (f *fsm) VerifyStateTransactions(transactions []*types.Transaction) error {
 }
 
 // Insert inserts the sealed proposal
-func (f *fsm) Insert(p *pbft.SealedProposal) error {
+func (f *fsm) Insert(proposal []byte, committedSeals []*messages.CommittedSeal) error {
+	// what to do with this?
+	newBlock := &types.Block{}
```

Comment:
I don't think we need this.

Reply:
We have removed `proposal` and `block` from `fsm`. That is way how `go-ibft` impl should work  https://github.com/0xPolygon/polygon-edge/commit/f0d08eb1a44c7b7b182c3f1757dc2e855bee7d57